go/vt/sqlparser/ast.go

```diff
@@ -4515,6 +4551,7 @@ type ColIdent struct {
 	// last field in the struct.
 	_            [0]struct{ _ []byte }
 	val, lowered string
+	at           AtCount
```

Comment:
What's the point of this?

Reply:
I was kinda hoping you would know, I just used vitessio/vitess's check constraint as a reference. I can remove the @ and @@ handling if you don't think it's relevant.